app/helpers/spotlight/main_app_helpers.rb

```diff
@@ -37,17 +37,24 @@ def presenter(document)
       end
     end
 
-    def exhibit_stylesheet_link_tag(tag)
-      if current_exhibit_theme && current_exhibit.theme != 'default'
-        stylesheet_link_tag "#{tag}_#{current_exhibit_theme}"
+    def themed_stylesheet_link_tag(tag)
```

Comment:
Similarly the method name change here would be breaking. Could you provide backwards compatibility and deprecation warnings?